templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -275,6 +282,11 @@ contents:
       echo "ERROR: Failed to activate ovs-if-br-ex NM connection"
       # if we made it here networking isnt coming up, revert for debugging
       set +e
+      # Revert back bond and vlan interfaces old connection
```

Comment:
i dont think we need to revert, i think we are just going to get ourselves in trouble if the script bails out mid run and doesnt revert the change. IMO we should just set device name

Reply:
removed the revert logic